### PR TITLE
Breadcrumb

### PR DESCRIPTION
**Pull request by mnuhell**
breadcrumb Is added with the FoundationPress style and the option GetText for the translations.

### CODE REVIEW

File: library/breadcrumb.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+function the_breadcrumb() {
+    global $post;
+    echo '<ul class="breadcrumbs" role="menubar" aria-label="breadcrumbs">';
```
Comment (themestechnology) [inline library/breadcrumb.php]: Need to add this into the ul for SEO/Schema 

itemscope="itemscope" itemtype="http://schema.org/WebPageElement"

**Comment by mvillagordo** [comment]
How can I delete the commits, now? or is better do a new pull request?

Thanks!!

**Comment by themestechnology** [comment]
Good work with that. But some people may not want the breadcrumb in the header, some would prefer the footer,

Do you think it would be worth me doing some code that would give the user the option, this could be including in the WP Customizer interface as a tick box?

**Comment by mvillagordo** [comment]
@themestechnology If you only want the breadcrumb in the footer, you only have copy the call function where you want.

```
<?php the_breadcrumb(); ?>
```

**Comment by FlatspinZA** [comment]
What @themestechnology was trying to suggest was plugin functionality to allow users to determine the location of the breadcrumbs through the WP admin panel, without tinkering with PHP files.

Dude, anything you can do to make this platform more awesome than it already is, would be greatly appreciated. 

I think this platform has awesome potential to become one of the best out there, and the more contributors, the better.

I was considering an SQL-driven solution to the breadcrumbs, because I think breadcrumbs are important. Unfortunately, I don't have time to dedicate to the things I want to do because I have just started a very time-taxing job.

**Comment by FlatspinZA** [comment]
Oh, thanks for the work, @mvillagordo ;)

**Comment by mvillagordo** [comment]
Forgive me for the pull request.

**Pull request closed without merging**